zim/main/__init__.py

```diff
@@ -39,12 +38,14 @@
 from .ipc import dispatch as _ipc_dispatch
 from .ipc import start_listening as _ipc_start_listening
 
+from typing import Optional, Tuple
+
 
 class HelpCommand(Command):
 	'''Class implementing the C{--help} command'''
 
 	usagehelp = '''\
-usage: zim [OPTIONS] [NOTEBOOK [PAGE]]
+usage: zim [OPTIONS] [NOTEBOOK [PAGE [ANCHOR]]]
```

Comment:
I would strongly prefer the ANCHOR argument to be provided via an option. I would suggest "--id" as the user interface also refers to these as "id" rather than "anchor".  

I'm OK with allowing the anchor as part of the PAGE argument, since the "#" is reserved and not allowed in page names.